### ISSUE TITLE
Uninstalling should clean-up `__groupfolders` folder

### ISSUE THREAD

**Issue by BloodyIron** [nextcloud/groupfolders#110]
I just removed (not just disabled, REMOVED) the Group Folders app from my nextCloud 13 lab, and the "__groupfolders" folder still exists. But it is in-fact empty.

This should be removed when removing the app, at least when the folder is completely empty.

**Comment by fschrempf** [comment]
Does it really hurt if the empty `__groupfolders` root directory on the filesystem remains after the app was deleted?

**Comment by BloodyIron** [comment]
Any application when being removed should properly clean up after itself. Leaving crap behind adds to administrative overhead and uncertainty whether it's necessary or not. Why leave it behind? It serves no purpose.

**Comment by fschrempf** [comment]
> Any application when being removed should properly clean up after itself. Leaving crap behind adds to administrative overhead and uncertainty whether it's necessary or not. Why leave it behind? It serves no purpose.

Yes, I agree with you. If the directory is empty and the app is removed, the directory should be removed, too.

**Comment by pierreozoux** [comment]
Relates to https://github.com/nextcloud/groupfolders/issues/272
